.github/workflows/commit.yml

```diff
@@ -144,51 +142,33 @@ jobs:
           pip install .
           pip install ".[test, optional]"
 
-      # Caching disabled until hashFiles issue resolved: https://github.com/actions/runner/issues/1840
-      # - name: Cache Modflow executables
-      #   uses: actions/cache@v3
-      #   with:
-      #     path: $HOME/.local/bin
-      #     key: ${{ runner.os }}-${{ hashFiles('flopy/utils/get_modflow.py') }}
-      #     restore-keys: |
-      #       ${{ runner.os }}-${{ hashFiles('flopy/utils/get_modflow.py') }}
+      - name: Extract home dir
+        id: get_home_dir
+        run: |
+          echo "::set-output name=home_dir::$HOME"
 
       - name: Install Modflow executables
-        run: |
-          mkdir -p $HOME/.local/bin
-          get-modflow $HOME/.local/bin
-          echo "$HOME/.local/bin" >> $GITHUB_PATH
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        uses: ./.github/actions/cache_exes
+        with:
+          path: ${{ steps.get_home_dir.outputs.home_dir }}/.local/bin
```

Comment:
There is probably a better way of doing this. The home path is different on ubuntu and mac runners, it seems nicer to evaluate it dynamically than hard-code in case the runner configurations ever change. I couldn't figure out how to evaluate `$HOME` and pass it as an action's input. Using it straight-forwardly doesn't evaluate the variable, it's just interpreted as a string. Accessing it like a user-provided environment variable e.g. `path: ${{ env.HOME }}/.local/bin` didn't work.

Reply:
Solved with tilde expansion